### ISSUE TITLE
Error on writing unavailable path in $importContent is not understandable.

### ISSUE THREAD

**Issue by saurabhdaware** [abelljs/abell#46]
**Describe the bug**
Hi, so like we care about what works, in static-site-generators we will also have to care about what doesn't work. And when something doesn't work, there should be a proper error message that mentions why it is not working. 

**To Reproduce**
In any .abell project, go to `[$path]/index.abell` and write `$importContent(undefined + '/index.md')`.

This will throw an error which is completely not understandable to users.

**Expected behavior**
It is supposed to show a better message that explains what is need to be changed.

**Additional context**

We can add try catch around markdown reading in `importAndRender` function in [src/utils/build-utils.js](https://github.com/abelljs/abell/blob/main/src/utils/build-utils.js)

**Comment by judicaelandria** [comment]
Hey,
can you guide a little bit and I will take this issue ?

**Comment by saurabhdaware** [comment]
Hi, so if we do a normal setup
```sh
npx create-abell-app my-blog
```

And in that if you go to `my-blog/theme/[$path]/index.abell`, You will find a line `{{ $importContent(meta.$path + '/index.md') }}`

If we build this, it will be built properly but for some reason, lets say meta.$path is undefined or someone writes `meta.path` instead of `meta.$path`, in that case it throws an error.

This error does not exactly tell what is wrong, rather it tries to read from `undefined/index.md` and the read fails. 

Instead of this not understandable error, we want an error like
```
Error in {{ $importContent() }} 
at [$path]/index.abell

content does not exist
```

**Comment by saurabhdaware** [comment]
https://github.com/abelljs/abell/blob/main/src/utils/build-utils.js#L169

This is the line that imports content

**Comment by judicaelandria** [comment]
so I just add a try catch method and add the custom error message ?

**Comment by saurabhdaware** [comment]
yes and if you feel like adding some more information that can help in finding where the problem is, you can go ahead and do that

**Comment by judicaelandria** [comment]
I finished it but how can I test if it work ?

**Comment by judicaelandria** [comment]
here is how I did it, I check if the contentPath is not undefined if it's undefined I return a error

**Comment by saurabhdaware** [comment]
- Do `npm link`, This will make `abell` command accessible to you from command line,
- In `examples/main/theme/[$path]/index.abell`, you can change the importContent path to `$importContent(undefined + '/index.md')`
- `abell build` inside `examples/main` to build the main project and check what it outputs
``

**Comment by judicaelandria** [comment]
![s](https://user-images.githubusercontent.com/46365844/87221265-45ec7080-c373-11ea-8731-3e4b4b87a35b.PNG)
is it good for you or I should change something ?

**Comment by saurabhdaware** [comment]
yes looks great!

**Comment by saurabhdaware** [comment]
Oh it is supposed to be error in $importContent

**Comment by judicaelandria** [comment]
oh yeah I'll change it and send a PR

**Comment by saurabhdaware** [comment]
Resolved in #48 Thanks!
